numbat/Cargo.toml

```diff
@@ -29,6 +29,8 @@ libc = "0.2.152"
 rust-embed = { version = "8.2.0", features = ["interpolate-folder-path"] }
 num-format = "0.4.4"
 walkdir = "2"
+chrono = "0.4.31"
+chrono-tz = "0.8.5"
```

Comment:
Unfortunately, this has a quite drastic effect on the WASM size.

Before: 872.45 KB (872,448 bytes)
After: 1.89 MB (1,892,352 bytes)

I'm not saying this is a no-go, but maybe we could look into reducing the size of those dependencies  (via feature flags?).

Reply:
Unfortunately I wasn't able to reduce the WASM size, even when disabled default features.  chrono-tz has a way to [limit the size](https://github.com/chronotope/chrono-tz?tab=readme-ov-file#limiting-the-timezone-table-to-zones-of-interest) of the TZ table which we might consider, but I'm not quite sure what ones we might omit (and I'm not currently sure how much space that would save anyway, compared to the size of chrono).   It also recommends using LTO, but we're already using that in our release profile.